### ISSUE TITLE
countrySelectComponent property ignored

### ISSUE THREAD

**Issue by BATCOH** [catamphetamine/react-phone-number-input#229]
Hello!

I'm trying to use the property `countrySelectComponent` with custom `Select` component
Minimal example:
```javascript
import React from "react";
import PhoneInput from "react-phone-number-input/mobile";

class CustomCountrySelect extends React.Component {
  render() {
    const { options, ...props } = this.props;
    return (
      <select {...props}>
        {options.map(({ value, label }) => (
          <option key={value} value={value}>
            {label}
          </option>
        ))}
      </select>
    );
  }
}

export default class extends React.Component {
  state = {
    phone: ""
  };
  render() {
    return (
      <PhoneInput
        countrySelectComponent={CustomCountrySelect}
        value={this.state.phone}
        onChange={phone => this.setState({ phone })}
      />
    );
  }
}
```
But [somewhere](https://github.com/catamphetamine/react-phone-number-input/blob/676417a1417f905cd43cc2d4ae098e3eb2b10f80/source/PhoneInputNative.js#L24) `countrySelectComponent` overwritten with `CountrySelectNative` and custom property just ignored.

Am I misundersteand docs and there be another way to add custom `countrySelectComponent`?

**Comment by catamphetamine** [comment]
Indeed, docs seem to be not correct.
Try `import { PhoneInput } from 'react-phone-number-input'`

**Comment by catamphetamine** [comment]
Or better: 
`import PhoneInput from 'react-phone-number-input/core'`

**Comment by catamphetamine** [comment]
Released `react-phone-number-input@2.3.3` with the fix.
Use it with `import PhoneInput from "react-phone-number-input/mobile";` as you originally did.

**Comment by BATCOH** [comment]
Thank you!
